src/webaudio.js

```diff
@@ -11,12 +11,12 @@ WaveSurfer.WebAudio = {
     },
 
     getAudioContext: function () {
-        if (!this.ac) {
-            this.ac = new (
+        if (!window.WaveSurfer.WebAudio.audioContext) {
```

Comment:
You don't need `window` here, since `WaveSurfer.WebAudio` is defined in the same file. It should not necessarily be a global object.